crates/subspace-farmer/src/plot.rs

```diff
@@ -184,6 +184,42 @@ pub struct Plot {
 }
 
 impl Plot {
+    /// Helper function for ignoring the error that given file/directory does not exist.
```

Comment:
Please move to the end of the impl, it makes sense to start impl with creation method instead.